src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md

```diff
@@ -127,37 +188,46 @@ The file which implements the cron-like process is defined in `cron.php`:
   while(true) {
     echo "[CRON] Running tasks\n";
     $report = $cron->run();
+
     while ($cron->isRunning()) { }
 
     echo "[CRON] " . count($report->getReports()) . " tasks have been executed\n";
+
     foreach($report->getReports() as $job_report) {
       $output = $job_report->getOutput();
       foreach($output as $line) {
         echo "[CRON] " . $line;
-      }
     }
+
     sleep(60);
   }
 ?>
 ```
 
-### Node.js
+#### Using Node.js
+
+The following packages may help you:
+- [node-cron](https://www.npmjs.com/package/cron)
+- [node-schedule](https://www.npmjs.com/package/node-schedule)
+
+##### Example Using `node-cron` Package
 
-In Node.js you can use different package such as [node-cron](https://www.npmjs.com/package/cron)
-or [node-schedule](https://www.npmjs.com/package/node-schedule).
+The code is available in our [sample-node-express](https://github.com/Scalingo/sample-node-express)
+repository.
 
-#### Example
+This example leverages the `node-cron` package to:
+- run `job1` every 2 minutes
+- run `job2` every minute
 
-The following example uses the `node-cron` package.
+Both jobs just log a message.
```

Comment:
best practice: we usually don't use words such as "just", "simply", etc in documentation pages

```suggestion
Both jobs log a message.
```